servicetest/testframework/lib.py

```diff
@@ -380,14 +380,22 @@ def __init__(self,
                  config_path=None,
                  caps_dir=None,
                  default_caps_dir=None,
-                 check_connection=True):
+                 check_connection=True,
+                 exec_prefix=None):
         if log_file_path is None:
             self.__log_file = subprocess.STDOUT
         else:
-            self.__log_file = open(log_file_path, "w")
+            if os.path.exists(log_file_path):
+                os.unlink(log_file_path)
```

Comment:
Hmm, the logs grow kind of quickly... the old behavior was to delete the file (default behavior of the write mode). 
I could add a flag though that defaults to deleting the file and if you want to keep it in a specific test...? but I don't see the point of it?